src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -338,16 +338,43 @@ def mul(self, y: Union[int, float, torch.Tensor]) -> "ReplicatedSharedTensor":
 
         return tensor
 
-    def truediv(self, y):
+    def truediv(self, y: Union[int, float, torch.Tensor]) -> "ReplicatedSharedTensor":
         """Apply the "div" operation between "self" and "y".
 
         Args:
-            y: self/y
+            y (Union[int, float, torch.Tensor]): Denominator.
```

Comment:
y should only be an int or torch.LongTensor or torch.Tensor right?

Reply:
Yeah , an error is raised when it is not int or tensor, in future when we add support for float,we could remove the sanity check.

Reply:
No I meant the type annotation.